sdk/js-query/src/query/request.ts

```diff
@@ -0,0 +1,70 @@
+import { BinaryWriter } from "./BinaryWriter";
+import { Network } from "./consts";
+import { utils } from "web3";
```

Comment:
I think we have web3 in the devDependencies, if we use it here we should promote it

Reply:
Good catch